project-base/app/src/DataFixtures/Demo/CustomerUserLoginTypeDataFixture.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\DataFixtures\Demo;
+
+use App\Model\Customer\User\CustomerUser;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Persistence\ObjectManager;
+use Override;
+use Shopsys\FrameworkBundle\Component\DataFixture\AbstractReferenceFixture;
+use Shopsys\FrontendApiBundle\Model\Customer\User\LoginType\CustomerUserLoginType;
+use Shopsys\FrontendApiBundle\Model\Customer\User\LoginType\CustomerUserLoginTypeDataFactory;
+use Shopsys\FrontendApiBundle\Model\Customer\User\LoginType\CustomerUserLoginTypeFacade;
+
+class CustomerUserLoginTypeDataFixture extends AbstractReferenceFixture implements DependentFixtureInterface
+{
+    /**
+     * @param \Shopsys\FrontendApiBundle\Model\Customer\User\LoginType\CustomerUserLoginTypeDataFactory $customerUserLoginTypeDataFactory
+     * @param \Shopsys\FrontendApiBundle\Model\Customer\User\LoginType\CustomerUserLoginTypeFacade $customerUserLoginTypeFacade
+     */
+    public function __construct(
+        protected readonly CustomerUserLoginTypeDataFactory $customerUserLoginTypeDataFactory,
+        protected readonly CustomerUserLoginTypeFacade $customerUserLoginTypeFacade,
+    ) {
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    #[Override]
+    public function load(ObjectManager $manager): void
+    {
+        $customerUser = $this->getReference(CustomerUserDataFixture::CUSTOMER_PREFIX . '1', CustomerUser::class);
+
+        $this->createWebLoginType($customerUser);
+        sleep(1); // to ensure that the second login type will have a different lastLoggedInAt value
```

Comment:
Why does it matter?

Reply:
I do not want to have two different logins at the same time for one user in the database (I know this sentence is funny, but this should never happen in reality). The functionality for getting the most recent login would be unreliable if there were two entries with the same time

Reply:
I do not want to have two different logins at the same time for one user in the database (I know this sentence is funny, but this should never happen in reality). The functionality for getting the most recent login would be unreliable if there were two entries at the same time